src/scene/scene.cpp

```diff
@@ -717,6 +717,20 @@ scripting::LuaLibrary Scene::luaLibrary() {
                 "Adds an interesting time to the current scene. The first argument is "
                 "the name of the time and the second argument is the time itself in the "
                 "format YYYY-MM-DDThh:mm:ss.uuu"
+            },
+            {
+                "worldPosition",
+                &luascriptfunctions::worldPosition,
+                {},
+                "string",
+                "Returns the worldPosition of the scene graph node with the given string as identifier"
+            },
+            {
+                "worldRotation",
+                & luascriptfunctions::worldRotation,
+                {},
+                "string",
+                "Returns the worldRotation of the scene graph node with the given string as identifier"
```

Comment:
Same here with `worldRotation`, and also mention that it is actually a matrix as rotations can be represented in different ways. So "world rotation matrix"